src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -476,6 +493,24 @@ public void setBackgroundMode(boolean backgroundMode) {
             }
         }
     }
+    public void setEnableScheduledScanJobs(boolean enabled) {
+        this.mScheduledScanJobsEnabled = enabled;
```

Comment:
I can throw an `IllegalStateException` if the BeaconService has already been started